main.js

```diff
@@ -24,3 +26,113 @@ window.onresize = () => {
+    images: 'images/desktop-switch/(3)Snapshoot desktop.png',
+    source: '#',
+    live: '#',
+  },
+
+  {
+    id: '4',
+    name: 'Uber Navigation',
+    year: '2015',
+    text: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry.Lorem Ipsum has, a galley of type and scrambled it 1960s. Lorem Ipsum is simplydummy text of the printing and typesetting industry.',
+    title: 'Uber',
+    stack: 'Lead Developer',
+    workStack: ['HTML', 'Ruby on Rails', 'CSS', 'Javascript'],
+    images: 'images/desktop-switch/(4)Snapshoot desktop.png',
+    source: '#',
+    live: '#',
+  },
+];
+
+if (portfolioItems) {
+  portfolioItems.forEach((cards) => {
+    cards.addEventListener('click', (e) => {
+      const idValue = e.target.closest('.card').id;
+      for (let i = 0; i < portfolioItemsDetail.length; i += 1) {
+        if (portfolioItemsDetail[i].id === idValue) {
+          const popupPage = document.createElement('page');
+          popupPage.classList.add('popup-page');
+          // console.log(porfolioItemsDetail[i].name);
+          popupPage.innerHTML = `
+          <div class="background"></div>
+          <div class="card flex-main">
+          <button class="close-btn" type="submit">
+          <i class="fas fa-times"></i>
+          </button>
+          <div class="card-right">
+              <h2>${portfolioItemsDetail[i].name}</h2>
+              <ul class="cano">
+                <li>${portfolioItemsDetail[i].title}<i class="fa fa-circle"></i></li>
+                <li>${portfolioItemsDetail[i].stack}<i class="fa fa-circle"></i></li>
+                <li>${portfolioItemsDetail[i].year}</li>
+              </ul>
+               <img src="${portfolioItemsDetail[i].images}" alt="${portfolioItemsDetail[i].name}">
+            <div class="flex-nl">
+              <p>${portfolioItemsDetail[i].text}</p>
+             
+              <ul class="lang">
+              <li><a href="#">HTML</a></li>
+              <li><a href="#">CSS</a></li>
+              <li><a href="#">Javascript</a></li>
+              </ul>
+              <div class="border"></div>
+              </div>       
+               <div class="link-button">
+                <button class="btn-default"><a href="${portfolioItemsDetail[i].live} class="see-btn">See live<img src="images/see-images/live.png" alt="live-link"/></a></button>
+                <button class="btn-default"><a href="${portfolioItemsDetail[i].source} class="see-btn">See source<img src="images/see-images/Vector.png" alt="live-link"/></a></button>
+               </div>
+             
+            </div>
+             
+            </div>
+          </div>
+          </div>
+          </div>
+          `;
+          wrapper.appendChild(popupPage);
+          const closeButton = popupPage.querySelector('.close-btn');
+          closeButton.addEventListener('click', () => {
+            wrapper.removeChild(popupPage);
+          });
+        }
+      }
+    });
+  });
+}
```

Comment:
- [ ] It is required to stick to the Figma design as close as possible. Your design for the popup window on both screen sizes are different from how they appear on the figma file. Kindly give a closer look into the figma designs and walk your designs alongside the figma design.

### Your Design Desktop

![your-popup-desktop](https://github.com/evansnyamekye/Portfolio-CompleteWebsite/assets/26597432/d9effab3-5cbe-4fb7-be18-f0582e245f24)


### Figma Design Desktop

![temp-popup-desktop](https://github.com/evansnyamekye/Portfolio-CompleteWebsite/assets/26597432/511184e1-2b59-4eb1-81c0-f8063ffc9475)


### Your Design Mobile

![your-popuu-mobile](https://github.com/evansnyamekye/Portfolio-CompleteWebsite/assets/26597432/fa9343fa-d298-4fe6-9063-31979c18ddd5)


### Figma Design Mobile

![temp-popup-mobile](https://github.com/evansnyamekye/Portfolio-CompleteWebsite/assets/26597432/e878d636-b5b8-4bb7-8fa4-6cc64d7f3eaa)